src/Cmixin/Holidays/gb-engwales.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+return array(
+    'new-year'      => function ($year) {
+        $date = DateTime::createFromFormat('U', (string) strtotime("$year/01/01"));
+
+        if (date('N', strtotime("$year/01/01")) == 6) {
+            $date->add(new DateInterval('P2D'));
```

Comment:
`$date->modify('+2 days');` would be more explicit and avoid relying `DateInterval`.